### PR TITLE
A few missing items for serialization/deserialization

### PR DESCRIPTION
**Pull request by MackeyK24**
Added ‘metadata’ to camera, light and mesh. Also added a few small
missing properties to make deserialization smoother.

### CODE REVIEW

File: Exporters/3ds Max/BabylonExport.Entities/BabylonCamera.cs [hunk 1]
```diff
@@ -1,84 +1,92 @@
...
+            rotation = new[] { 0f, 0f, 0f };
+
+            // Default values
+            fov = 0.8f;
+            minZ = 0.1f;
+            maxZ = 5000.0f;
+            speed = 1.0f;
+            inertia = 0.9f;
+
+			metadata = null;
```
Comment (deltakosh) [inline Exporters/3ds Max/BabylonExport.Entities/BabylonCamera.cs]: Weird spacing

File: Exporters/3ds Max/BabylonExport.Entities/BabylonCamera.cs [hunk 2]
```diff
@@ -1,84 +1,92 @@
...
+        [DataMember]
+        public int autoAnimateTo { get; set; }
+
+        [DataMember]
+        public bool autoAnimateLoop { get; set; }
+
+        [DataMember]
+        public BabylonAnimation[] animations { get; set; }
+
+		[DataMember]
```
Comment (deltakosh) [inline Exporters/3ds Max/BabylonExport.Entities/BabylonCamera.cs]: Same

File: Exporters/3ds Max/BabylonExport.Entities/BabylonLight.cs [hunk 3]
```diff
@@ -1,76 +1,83 @@
...
+        [DataMember]
+        public int autoAnimateTo { get; set; }
+
+        [DataMember]
+        public bool autoAnimateLoop { get; set; }
+
+        [DataMember]
+        public BabylonAnimation[] animations { get; set; }
+
+		[DataMember]
```
Comment (deltakosh) [inline Exporters/3ds Max/BabylonExport.Entities/BabylonLight.cs]: Here as well

**Comment by MackeyK24** [comment]
I think because that was a C# project it opened up in Xamarin Studio For Mac... Must be unix,line endings or something like that... Now i stink at github... Im just gonna DELETE and RE-FORK to grab a fresh copy and make the changes again using VSCode... Which is what i have been using for the typescript mods i submitted.

Did you get my question about the BABYLON.ShaderMaterial and BABYLON.HDRCubeTexture classed... Did you look at em... do you me to PR this classes as well?

**Comment by deltakosh** [comment]
Makes sense
Please go ahead and add metadata where you need it

**Pull request closed without merging**